lib/ui/serverlessFunctionLogs.ts

```diff
@@ -14,36 +13,63 @@ const LOG_STATUS_COLORS = {
   HANDLED_ERROR: Styles.error,
 };
 
-function errorHandler(log, options) {
+type LogStatus = keyof typeof LOG_STATUS_COLORS;
+
+type Log = {
+  status: LogStatus;
+  createdAt: string;
+  executionTime: number;
+  log?: string;
+  error?: {
+    type: string;
+    message: string;
+    stackTrace?: string[];
+  };
+};
+
+type LogsResponse = {
+  results?: Log[];
```

Comment:
Is this supposed to always be defined?  We have a type predicate `isLogsResponse` that is checking that this is defined